### PR TITLE
first pass at a logfile to github issue tool

### PR DESCRIPTION
**Pull request by jkrauska**
* walks list of log crashes
* checks for dupes locally
* asks user if they want to submit an issue
* opens a new issue with crash details
* FIXME: figure out how to attach or reference a confdir exception file

### CODE REVIEW

File: scripts/logcrash-to-github.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+                    else:
+                        seen_exns.append(exn)
+
+                    print('-'*80)
+                    print(crashdir)
+                    print('New: %s' % exn)
+                    print(json.dumps(data, indent=4, sort_keys=True))
+                    if yes_or_no('Create new issue?'):
+                        # FIXME - how to attach gz to issue.
+                        title = 'TESTING - CRASH - TESTNET - %s' % exn.strip()
```
Comment (bkase) [inline scripts/logcrash-to-github.py]: assume you want to update this before merge the pr yeah?

Comment (jkrauska) [reply scripts/logcrash-to-github.py]: the bit about testing?

I'm ok leaving it -- I want it to be clear that the Issue tool isn't perfect yet. 
It's trivial to rename the title in github.

Comment (jkrauska) [reply scripts/logcrash-to-github.py]: or the fixme: ?

main problem is that github API doesn't allow you to upload attachments. (that I could find)

@conner thinks it's a private api and I was going to see if he could figure out how to call it.

other methods would be to upload object to s3, and reference it here.

Script provides value today as it creates an issue with some of the metadata needed.

**Comment by jkrauska** [comment]
Q: Should we also include hostname so staff could login?

**Pull request merged**